validator/client/validator.go

```diff
@@ -1140,6 +1140,10 @@ func (v *validator) buildSignedRegReqs(ctx context.Context, pubkeys [][fieldpara
 		feeRecipient := common.HexToAddress(params.BeaconConfig().EthBurnAddressHex)
 		gasLimit := params.BeaconConfig().DefaultBuilderGasLimit
 		enabled := false
+		registrationTimestamp := uint64(time.Now().UTC().Unix())
```

Comment:
i don't think it makes sense to add it here in the loop I think either outside or right before the `&ethpb.ValidatorRegistrationV1` makes more sense to me and then you can reuse that variable for that struct.
I don't think this will likely happen but did you think about having partial registrations being better in some way?